src/components/cvaa-overlay/cvaa-overlay.tsx

```diff
@@ -144,7 +144,13 @@ class CVAAOverlay extends Component<any, any> {
         open
         handleKeyDown={this.props.handleKeyDown}
         addAccessibleChild={this.props.addAccessibleChild}
-        onClose={props.onClose}
+        onClose={() => {
+          props.onClose();
+          setTimeout(() => {
```

Comment:
how do you know that a timeout of 0 would be enough ? is there a condition you can check ?
also note that we have a util function in ui that waits for an element to be visible before focusing
https://github.com/kaltura/playkit-js-ui/blob/2d97fd5fb385ad68f5a13012163d0daee93e560f/src/utils/focus-element.ts#L9